File: 0813/813.cs
public class Solution {
    public int LargestSumOfAverages(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(LargestSumOfAverages(root.left), LargestSumOfAverages(root.right));
    }
}

File: 0813/813.c
int largestSumOfAverages(struct TreeNode* root) {
    if (!root) return 0;
    int l = largestSumOfAverages(root->left);
    int r = largestSumOfAverages(root->right);
    return 1 + (l > r ? l : r);
}